src/Imagick/Image.php

```diff
@@ -848,6 +848,23 @@ private function applyImageOptions(\Imagick $image, array $options, $path)
                     }
                 }
                 break;
+            case 'jxl':
+                if (!empty($options[$format . '_lossless'])) {
+                    $image->setimagecompressionquality(100);
+                    $image->setcompressionquality(100);
+                } else {
+                    if (!isset($options[$format . '_quality'])) {
+                        if (isset($options['quality'])) {
+                            $options[$format . '_quality'] = $options['quality'];
+                        }
+                    }
+                    if (isset($options[$format . '_quality'])) {
+                        $options[$format . '_quality'] = max(9, min(99, $options[$format . '_quality']));
```

Comment:
Is the `9` here correct?

Reply:
I think so, yes. But I can’t seem to remember why now. Give me a second... ☺️

Reply:
> I think so, yes. But I can’t seem to remember why now. Give me a second... ☺️

I understand... sorry for the delay in reviewing this PR...

Reply:
Just tested it manually and qualities from 0 to 8 all create the exact same image with a size about five times larger than quality 9.

I think we should merge it as it is now and probably create a new ticket to check if this is a bug that needs to be fixed in ImageMagick.